lib/libze/libze.c

```diff
@@ -2036,6 +2036,52 @@ typedef struct libze_destroy_cbdata {
     libze_destroy_options *options;
 } libze_destroy_cbdata;
 
+
```

Comment:
nit: for code readability, I typically align the fields (makes pointers vs regular variables stand out), but up to you:

```
typedef struct dependents_cbdata {
    libze_destroy_cbdata    *cbd;
    zfs_handle_t            *cb_target;
    nvlist_t                *cb_nvl;
} dependents_cbdata_t;
```